queries/ruby/highlights.scm

```diff
@@ -45,8 +45,8 @@
 
 ; Function calls
 
-((identifier) @function
- (#eq? @function "require"))
+((identifier) @include
+ (#vim-match? @include "^(require|require_relative|load)$"))
```

Comment:
I don't really think it is worth using vim-match here
```suggestion
 (#lua-match? @include "^(require|require_relative|load)$"))
```

Reply:
lua does not support `|`

Reply:
Huh, you are right, I still need to work on my lua-fu